.github/workflows/ci.yml

```
@@ -0,0 +1,69 @@
+          python -m pip install --upgrade pip
+          pip install -r requirements.txt || pip install .
+          pip install flake8 pytest black isort
+      
+      - name: Lint with flake8
+        run: |
+          flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
+          flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
+      
+      - name: Format check with black
+        run: black --check . || echo "Code formatting issues found"
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)